README.md

```diff
@@ -1,25 +1,128 @@
-# Getting Started with Create React App
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you add your own logo **(Not Microverse logo)** at the top of your readme file to make yourself a brand in the tech industry to catch the eyes of recruiters 👍